### DISCUSSION TITLE
Disable command history

### DISCUSSION THREAD

**Discussion by avidseeker** [cmus/cmus#1284] [General]
Is that possible? a workaround I can think of is:

```bash
echo > ~/.config/cmus/command-history
chmod 400 ~/.config/cmus/command-history
```

Edit: same question for ~/.config/cmus/search-history

**Comment by ilf** [comment]
I would also like to disable command and search history.
I tried symlinking `command-history` and `search-history` to `/dev/null`, but cmus changes them to files again.
